README.md

```diff
@@ -14,11 +14,9 @@ Modelled as a state machine, with the help of AWS step functions, Trapheus resto
   <a href="https://github.com/intuit/Trapheus/releases"><img src="https://img.shields.io/github/v/release/intuit/trapheus.svg" alt="release badge"/></a>
 </p>
 
- <img src="https://ch-resources.oss-cn-shanghai.aliyuncs.com/images/lang-icons/icon128px.png" width="22px" /> [English](README.md) |  [简体中文](./docs/README.zh-CN.md) | [français](./docs/README.fr.md)
-
-
-* **Important:** this application uses various AWS services and there are costs associated with these services after the Free Tier usage - please see the [AWS  pricing page](https://aws.amazon.com/pricing/) for details.
+<img src="https://ch-resources.oss-cn-shanghai.aliyuncs.com/images/lang-icons/icon128px.png" width="22px" /> [English](README.md) | [简体中文](./docs/README.zh-CN.md) | [français](./docs/README.fr.md) | [español](./docs/README.es.md)
 
+- **Important:** this application uses various AWS services and there are costs associated with these services after the Free Tier usage - please see the [AWS pricing page](https://aws.amazon.com/pricing/) for details.
```

Comment:
can we remove all the unrelated changes from this file. @virejdasani your required changes seem to be in this line https://github.com/intuit/Trapheus/pull/158/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R17 , would my understanding be correct?